tensorflow_lite_support/python/task/processor/proto/segmentations_pb2.py

```diff
@@ -13,7 +13,215 @@
+    used.
+
+  Attributes:
+    value: The value indicates the prediction confidence usually in the
+      range [0, 1].
+  """
+  value: np.ndarray
+
+  @doc_controls.do_not_generate_docs
+  def to_pb2(self) -> _ConfidenceMaskProto:
+    """Generates a protobuf object to pass to the C++ layer."""
+    return _ConfidenceMaskProto(value=self.value)
+
+  @classmethod
+  @doc_controls.do_not_generate_docs
+  def create_from_pb2(cls, pb2_obj: _ConfidenceMaskProto) -> "ConfidenceMask":
+    """Creates a `ConfidenceMask` object from the given protobuf object and
+    the segmentation mask's width and height."""
+    return ConfidenceMask(value=np.array(pb2_obj.value))
+
+  def __eq__(self, other: Any) -> bool:
+    """Checks if this object is equal to the given object.
+    Args:
+      other: The object to be compared with.
+    Returns:
+      True if the objects are equal.
+    """
+    if not isinstance(other, ConfidenceMask):
+      return False
+
+    return self.to_pb2().__eq__(other.to_pb2())
+
+
+@dataclasses.dataclass
+class ColoredLabel:
+  """Defines a label associated with an RGB color, for display purposes.
+
+  Attributes:
+    color: The RGB color components for the label, in the [0, 255] range.
+    category_name: The class name, as provided in the label map packed in the
+      TFLite ModelMetadata.
+    display_name: The display name, as provided in the label map
+      (if available) packed in the TFLite Model Metadata .
+  """
+
+  color: Tuple[int, int, int]
+  category_name: str
+  display_name: str
+
+  @doc_controls.do_not_generate_docs
+  def to_pb2(self) -> _ColoredLabelProto:
+    """Generates a protobuf object to pass to the C++ layer."""
+    r, g, b = self.color
+    return _ColoredLabelProto(r=r, g=g, b=b,
+                              class_name=self.category_name,
+                              display_name=self.display_name)
+
+  @classmethod
+  @doc_controls.do_not_generate_docs
+  def create_from_pb2(cls, pb2_obj: _ColoredLabelProto) -> "ColoredLabel":
+    """Creates a `ColoredLabel` object from the given protobuf object."""
+    return ColoredLabel(color=(pb2_obj.r, pb2_obj.g, pb2_obj.b),
+                        category_name=pb2_obj.class_name,
+                        display_name=pb2_obj.display_name)
+
+  def __eq__(self, other: Any) -> bool:
+    """Checks if this object is equal to the given object.
+    Args:
+      other: The object to be compared with.
+    Returns:
+      True if the objects are equal.
+    """
+    if not isinstance(other, ColoredLabel):
+      return False
+
+    return self.to_pb2().__eq__(other.to_pb2())
+
+
+@dataclasses.dataclass
+class Segmentation:
+  """Represents one Segmentation object in the image segmenter's results.
+
+  Attributes:
+    colored_labels: A list of `ColoredLabel` objects.
+    category_mask: A flattened NumPy 2D-array of the category mask.
+    confidence_masks: A list of `ConfidenceMask` objects.
+  """
```

Comment:
You should include width and height values here.
https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/cc/task/vision/proto/segmentations.proto#L82